File: ex02/ex02_03/ex02_03.js
let globalNum = 100;

function printNums() {
  let functionNum = 200;
  console.log("Integer in global: " + String(globalNum));
  console.log("Integer in function: " + String(functionNum));
}

function main() {
  console.log("Initial Integer in global: " + String(globalNum));
  globalNum = 101; 
  printNums();
  {
    let blockNum = 300;
    console.log("Integer in block: " + String(blockNum));
  }
  // blockNum = 301;
  let functionNum = 102;
  return 0;
}

main();

File: ex02/ex02_03/ex02_03.cpp
#include <iostream>

auto global_num = 100;

auto print_nums() {
  auto function_num = 200;
  std::cout << "Integer in global: " << global_num << std::endl;
  std::cout << "Integer in function: " << function_num << std::endl;
}

int main() {
  std::cout << "Initial Integer in global: " << global_num << std::endl;
  global_num = 101; 
  print_nums();
  while (1) {
    int block_num = 300;
    std::cout << "Integer in block: " << block_num << std::endl;
    break;
  }
  // block_num = 301;
  // function_num = 102;
  return 0;
}